### ISSUE TITLE
lsp-java-boot doesn't show lsp-ui correctly, with error: wrong-type-argument plistp

### ISSUE THREAD

**Issue by RobinBeekhof** [emacs-lsp/lsp-java#294]
**Describe the bug**
I added lsp-java in a new install of spacemacs (dev branch). lsp-java itself works fine and shows lsp-ui. However when I add lsp-java-boot lsp-ui doesn't work anymore and I get the error "Error processing message (wrong-type-argument plistp..."

**Expected behavior**
I expect lsp-ui to be shown correctly, for example:
![image](https://user-images.githubusercontent.com/50193227/99403003-47621280-28ea-11eb-8553-f53ac87cc73c.png)

**Screenshots**
I added java-lsp to dotspacemacs-configuration-layers
![image](https://user-images.githubusercontent.com/50193227/99399497-08ca5900-28e6-11eb-954b-433022837e5a.png)

I added the config for lsp-java-boot to the user-config
![image](https://user-images.githubusercontent.com/50193227/99399213-a6715880-28e5-11eb-9bb0-bb6d57cfeafc.png)

**Logs**
Spacemacs is ready.
Skipping check for new version (reason: dotfile)
Package cl is deprecated
[yas] Prepared just-in-time loading of snippets successfully.
using config for config_mac
LSP :: Waiting for connection for boot-ls, retries: 0
LSP :: Waiting for connection for boot-ls, retries: 1
LSP :: Waiting for connection for boot-ls, retries: 2
LSP :: Waiting for connection for boot-ls, retries: 3
LSP :: jdtls:26321 initialized successfully
Error running timer: (error "The connected server(s) does not support method textDocument/documentHighlight.
To find out what capabilities support your server use ‘M-x lsp-describe-session’
and expand the capabilities section")
LSP :: Waiting for connection for boot-ls, retries: 4
LSP :: Waiting for connection for boot-ls, retries: 5
LSP :: Waiting for connection for boot-ls, retries: 6
LSP :: Waiting for connection for boot-ls, retries: 7
LSP :: Waiting for connection for boot-ls, retries: 8
LSP :: Waiting for connection for boot-ls, retries: 9
LSP :: Waiting for connection for boot-ls, retries: 10
LSP :: Waiting for connection for boot-ls, retries: 11
LSP :: Waiting for connection for boot-ls, retries: 12
LSP :: Waiting for connection for boot-ls, retries: 13
LSP :: Waiting for connection for boot-ls, retries: 14
LSP :: Waiting for connection for boot-ls, retries: 15
LSP :: Successfully connected to boot-ls
LSP :: Connected to [jdtls:26321][boot-ls:26323 status:starting].
LSP :: boot-ls:26323 initialized successfully
End of buffer [6 times]
Search failed: there is an unmatched expression somewhere or we are at the beginning/end of file. [4 times]
Error processing message (wrong-type-argument plistp (#s(hash-table size 2 test equal rehash-size 1.5 rehash-threshold 0.8125 data ("language" "java" "value" "java.lang.Override")) "Indicates that a method declaration is intended to override a method declaration in a supertype. If a method is annotated with this annotation type compilers are required to generate an error message unless at least one of the following conditions hold:

**Comment by yyoncho** [comment]
most likely lsp-mode bug related to merging the information from both servers.

**Comment by RobinBeekhof** [comment]
Do you have a suggestion how I could fix this?

**Comment by RobinBeekhof** [comment]
Yes great! this resolved the errors. I do notice now however that lsp-ui does not show the pop-up when both lsp-java and lsp-java-boot are active in the same file. Do I need to do some config to enable this? I'm still quite new to emacs/spacemacs

**Comment by yyoncho** [comment]
I have to investigate the lsp-ui issue. Setting lsp-ui-doc-include-signature  to t might fix the issue.

**Comment by RobinBeekhof** [comment]
I added (setq lsp-ui-doc-include-signature t) to my user-config, but that did not make any difference

**Comment by yyoncho** [comment]
Then it seems like it is lsp-mode/lsp-ui bug.

**Comment by catamorphisms** [comment]
I am experiencing the same issue

**Comment by CaderHancock** [comment]
I would also love a resolution to this issue.

**Comment by yyoncho** [comment]
I will merge https://github.com/yyoncho/lsp-mode/commit/06c5677a067666ba140451606040405fa972ce64 later today.
